src/autopas/tuning/tuningStrategy/TuningStrategyInterface.h

```diff
@@ -50,9 +53,12 @@ class TuningStrategyInterface {
    *
    * @param configQueue Queue of configurations to be tested. The tuning strategy should edit this queue.
    * @param evidenceCollection All collected evidence until now.
+   * @param intentionalConfigWipe Optional flag that is used by tuning strategies to signal they have intentionally
+   * wiped the config queue
    */
-  virtual void optimizeSuggestions(std::vector<Configuration> &configQueue,
-                                   const EvidenceCollection &evidenceCollection) = 0;
+  virtual void optimizeSuggestions(
+      std::vector<Configuration> &configQueue, const EvidenceCollection &evidenceCollection,
+      std::optional<std::reference_wrapper<bool>> intentionalConfigWipe = std::nullopt) = 0;
```

Comment:
Why is the parameter a `std::optional` that holds a reference? Why not a reference to a `std::optional`? That would also save us a copy. Or does this not work with the default parameter?

I'm not a huge fan of in/out parameters like these in general as well as default parameters on virtual methods :/ Why not simply return the object or just true / false?

Reply:
Changed this out-parameter to a bool return type in [976b882](https://github.com/AutoPas/AutoPas/pull/936/commits/976b88217759053e644121139c2901b725f69ffc)